Makefile

```diff
@@ -151,11 +155,11 @@ ifeq (1, $(PREP_DOCKER))
 	make docker-prep
 endif
 	($(SA) $(ENV) && GATEWAY_HOST=$(ITEST_HOST) KERNEL_USERNAME=$(ITEST_USER) nosetests -v enterprise_gateway.itests)
-	@echo "Run \`docker logs itest\` to see enterprise-gateway log."
+	@echo "Run \`docker logs $(ITEST_HOSTNAME_PREFIX)\` to see enterprise-gateway log."
 
 PREP_TIMEOUT?=60
 docker-prep: 
-	@-docker rm -f itest >> /dev/null
+	@-docker rm -f $(ITEST_HOSTNAME_PREFIX) >> /dev/null
 	@echo "Starting enterprise-gateway container (run \`docker logs itest\` to see container log)..."
-	@-docker run -itd -p 8888:8888 -h itest --name itest -v `pwd`/enterprise_gateway/itests:/tmp/byok elyra/enterprise-gateway-demo:$(ENTERPRISE_GATEWAY_TAG) --elyra
-	@(r="1"; attempts=0; while [ "$$r" == "1" -a $$attempts -lt $(PREP_TIMEOUT) ]; do echo "Waiting for enterprise-gateway to start..."; sleep 2; ((attempts++)); docker logs itest |grep 'Jupyter Enterprise Gateway at http'; r=$$?; done; if [ $$attempts -ge $(PREP_TIMEOUT) ]; then echo "Wait for startup timed out!"; exit 1; fi;)
+	@-docker run -itd -p 8888:8888 -h $(ITEST_HOSTNAME_PREFIX) --name $(ITEST_HOSTNAME_PREFIX) -v `pwd`/enterprise_gateway/itests:/tmp/byok elyra/enterprise-gateway-demo:$(ENTERPRISE_GATEWAY_TAG) --elyra
+	@(r="1"; attempts=0; while [ "$$r" == "1" -a $$attempts -lt $(PREP_TIMEOUT) ]; do echo "Waiting for enterprise-gateway to start..."; sleep 2; ((attempts++)); docker logs $(ITEST_HOSTNAME_PREFIX) |grep 'Jupyter Enterprise Gateway at http'; r=$$?; done; if [ $$attempts -ge $(PREP_TIMEOUT) ]; then echo "Wait for startup timed out!"; exit 1; fi;)
```

Comment:
Sorry, but I find it confusing to use a "prefix" for the host and container name.  It just feels wrong and a bit clunky.  Instead, just add a -e option (e.g., `-e ITEST_HOSTNAME_PREFIX=${ITEST_HOSTNAME_PREFIX}`) to the docker run command, leaving the previous -h and --name options intact.